configs/katello/3.15.yaml

```diff
@@ -0,0 +1,57 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+:releases:
+  :3.15.0:
+    :redmine_version_id: 1131
+:code_name: Rocky Road
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.15
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.24
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.20.1
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.2
```

Comment:
This will be 3.5.3 once we release the next version, no harm in bumping it now I think